[rospy.client][INFO] 2021-11-29 10:51:32,687: init_node, name[/server_endpoint], pid[19464]
[xmlrpc][INFO] 2021-11-29 10:51:32,688: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-11-29 10:51:32,688: Started XML-RPC server [http://wng:39705/]
[rospy.init][INFO] 2021-11-29 10:51:32,688: ROS Slave URI: [http://wng:39705/]
[rospy.impl.masterslave][INFO] 2021-11-29 10:51:32,688: _ready: http://wng:39705/
[rospy.registration][INFO] 2021-11-29 10:51:32,688: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-11-29 10:51:32,688: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-11-29 10:51:32,788: registered with master
[rospy.rosout][INFO] 2021-11-29 10:51:32,788: initializing /rosout core topic
[rospy.rosout][INFO] 2021-11-29 10:51:32,789: connected to core topic /rosout
[rospy.simtime][INFO] 2021-11-29 10:51:32,790: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-11-29 10:51:32,799: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-11-29 10:51:32,993: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-11-29 10:51:42,632: Connection from 192.168.1.146
[rosout][INFO] 2021-11-29 10:51:42,636: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-11-29 10:51:42,737: topic[/execution_check] adding connection to [http://wng:39705/], count 0
[rospy.internal][INFO] 2021-11-29 10:51:42,738: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-11-29 11:11:24,287: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2021-11-29 11:11:24,291: Unable to initiate TCP/IP socket to wng:41693 (http://wng:39705/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-11-29 11:11:24,291: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-11-29 11:11:24,294: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-11-29 11:11:24,294: topic[/execution_check] removing connection to http://wng:39705/
[rospy.impl.masterslave][INFO] 2021-11-29 11:11:24,294: signal-2
[rospy.core][INFO] 2021-11-29 11:11:24,425: signal_shutdown [atexit]
